apps/playground-lit/src/app/app.element.ts

```diff
@@ -17,18 +17,24 @@ export class AppElement extends LitElement {
 
     render(): TemplateResult {
         return html`
-            <a is="vl-link" id="btn-acties">Acties</a>
-            <vl-popover data-vl-for="btn-acties" data-vl-placement="bottom-start">
-                <ul is="vl-link-list">
-                    <li is="vl-link-list-item">
-                        <a is="vl-link">Voeg gebruiker toe.</a>
-                    </li>
-                    <li is="vl-link-list-item">
-                        <a is="vl-link">Voeg adres toe.</a>
-                    </li>
-                </ul>
-            </vl-popover
-
+            <main>
+                <a is="vl-link" id="btn-acties">Acties</a>
+                <vl-popover for="btn-acties" placement="bottom-start">
+                    <vl-popover-action-list
+                        @click=${(event: CustomEvent) => {
+                            const actionElement = event.target as VlPopoverActionComponent;
+                            if (actionElement instanceof VlPopoverActionComponent) {
+                                // do action
+                                console.log('vl-popover-action clicked > ' + actionElement.action);
```

Comment:
logs laten we staan?

Reply:
ah is playground, never mind

Reply:
ja is gwn om te tonen hoe je eventueel iets met een actie zou kunnen doen

in ieder geval playgrounds zijn eerder voor ons bedoeld, maar willen daar ook geen slechte voorbeelden in tonen